Images_And_Graphics_Index1.html

```diff
@@ -96,9 +19,9 @@ <h1>Rangituhi Walk</h1>
 			
 		<div>
 	    <div class="header">
-			<img src="Images/Banner01.jpg" alt="Banner">
-			<a href="https://projectspace.nz/npbocvyj/2.4+2.8_Iterative_Build_LFarah/Stage3_Image_Graphics/Images_Graphics_v1/index.v1.html">
-			<img class="Image5" src="Images/Tawa_College_Logo1.jpg" alt="logo"></a>	
+			<img src="Images/3Rangituhi_Walk_View.jpg" alt="Banner">

```

Comment:
In my first version I added EOTC on the Header. I got feedback from Ayman Farah and he told me to add the Rangituhi walk view to make it look better.